### ISSUE TITLE
Instruction about drawing connections

### ISSUE THREAD

**Issue by alreadydone** [nomeata/incredible#102]
When I first tried IPM a few months ago, I tried to fit the dents of two blocks together, and didn't make any progress even on the very first problem (quite hilarious in hindsight), because I didn't realize that I need to connect the blocks. When I revisited it yesterday, I found out in a [referenced paper](https://www.joachim-breitner.de/publications/Incredible_ITP2016_preprint.pdf) what I am supposed to do. Since I am unlikely the only one to make this mistake, it would be the nice to just add Fig.1 in that paper to your instructions (shown by clicking "i" in the top-right corner on http://incredible.pm).

**Comment by nomeata** [comment]
Yeah, that UI problem has bit many. Do you want to provide a patch?

**Comment by alreadydone** [comment]
I'd like to but I am not sufficiently skilled in Javascript yet :)
